README.md

```diff
@@ -755,6 +755,15 @@ You can also see the help for the command in the CLI:
 $ affinidi verify-vc --help
 ```
 
+### **affinidi start**
+
+Also called `wizard`.
+
+This command provides you a guide from end to end, step by step, with selectable options.
```

Comment:
"This command provides you with..."
or better
"This command provides you with an end-to-end, step-by-step guide to the Affinidi CLI, with selectable options"